packages/rollup-utils/index.mjs

```diff
@@ -0,0 +1,9 @@
+Error.stackTraceLimit = Infinity;
```

Comment:
why do we need this?

Reply:
We already had this before. I believe this was so that we get bigger (ie not truncated) stack traces in case of an error.

Reply:
nice TIL

Reply:
This significantly affects the build performance.

Reply:
I can see that. Will remove. Stack traces during build are somewhat meh anyhow.